Bio/PDB/NeighborSearch.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, atom_list, bucket_size=10):
            with this to optimize speed if you feel like it.
 
         """
-        from Bio.PDB.kdtrees import KDTree
+        from Bio.PDB.kdtrees import KDTree  # type: ignore
```

Comment:
@JoaoRodrigues I don't see any `# notypes` in this PR, maybe you meant this? We can still add type hints to KDTree later, using a `KDTree.pyi` file

Reply:
That's what I meant, sorry !